package.json

```diff
@@ -52,7 +52,7 @@
     "nock": "^13.3.2",
     "nodemon": "^3.0.1",
     "p-limit": "^3.1.0",
-    "prettier": "^2.8.8",
+    "prettier": "^3.0.0",
```

Comment:
we are still pulling in older versions of prettier as transient dependencies. maybe we should wait until all other dependencies have been updated to use prettier 3.x?

Reply:
From I can tell, the older versions of Prettier are coming from `@storybook/cli` and `@storybook/codemod`

https://github.com/storybookjs/storybook/blob/377f7f9c0a022ae073ab0e93792d4e15dd88b15c/code/lib/cli/package.json#L87

https://github.com/storybookjs/storybook/blob/377f7f9c0a022ae073ab0e93792d4e15dd88b15c/code/lib/codemod/package.json#L60